src/components/Tabs/Tabs.test.tsx

```diff
@@ -21,6 +21,38 @@ test('it renders Tabs', async () => {
   expect(options.length).toEqual(2);
 });
 
+test('it renders variant pill Tabs', async () => {
+  const { findByTestId } = renderWithTheme(
+    <Tabs variant="pill">
+      <TabList aria-label="Test Tabs" data-testid={testId}>
+        <Tab stopId="tab1">Tab 1</Tab>
+        <Tab stopId="tab2">Tab 2</Tab>
+      </TabList>
+    </Tabs>
+  );
+
+  const root = await findByTestId(testId);
+  const options = root.querySelectorAll('[role="tab"]');
+  expect(options).toBeTruthy();
+  expect(options.length).toEqual(2);
+});
```

Comment:
Is there a way to check that the `pill` variant is applied somewhere in the DOM?  May be able to leverage `toHaveClass()` and assert on that.

Reply:
@nicolls1 can you take a look at updating per @ynotdraw's suggestion?

Reply:
Pushed